packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.test.ts

```diff
@@ -170,6 +170,84 @@ describe('GraphQLSitemapService', () => {
       ]);
     });
 
+    it('should return personalized paths when personalize data is returned', async () => {
+      const lang1 = 'ua';
```

Comment:
Just about naming convention, you have a single locale here, so it should be `lang`